### PR TITLE
Update Profiles in PList, PInstanceList & CONFIG

### PR DESCRIPTION
**Pull request by openBackhaul**

### CODE REVIEW

File: OperationKeyManagement+config.json [hunk 1]
```diff
@@ -11,14 +11,275 @@
...
+                },
+                "action-profile-configuration": {
+                  "request": {
+                    "protocol-reference": "/core-model-1-4:control-construct/logical-termination-point=okm-2-0-0-tcp-s-bm-001/layer-protocol=0/tcp-server-interface-1-0:tcp-server-interface-pac/tcp-server-interface-configuration/local-protocol",
+                    "address": {
+                      "ipv-4-address-reference": "/core-model-1-4:control-construct/logical-termination-point=okm-2-0-0-tcp-s-bm-001/layer-protocol=0/tcp-server-interface-1-0:tcp-server-interface-pac/tcp-server-interface-configuration/local-address/ipv-4-address"
+                    },
+                    "port-reference": "/core-model-1-4:control-construct/logical-termination-point=okm-2-0-0-tcp-s-bm-001/layer-protocol=0/tcp-server-interface-1-0:tcp-server-interface-pac/tcp-server-interface-configuration/local-port",
+                    "operation-reference": "/core-model-1-4:control-construct/logical-termination-point=okm-2-0-0-op-s-bs-005/layer-protocol=0/operation-server-interface-1-0:operation-server-interface-pac/operation-server-interface-capability/operation-name"
```
Comment (IswaryaaS) [inline OperationKeyManagement+config.json:77]: An LTP instance referring "/docs" (okm-2-0-0-op-s-bs-005) and its corresponding reference in client-ltp list of okm-1-0-0-http-s-0000 would need to be added in config.json file

File: OperationKeyManagement+config.json [hunk 2]
```diff
@@ -11,14 +11,275 @@
...
+            {
+              "uuid": "okm-2-0-0-response-p-007",
+              "profile-name": "response-profile-1-0:PROFILE_NAME_TYPE_GENERIC_RESPONSE_PROFILE",
+              "response-profile-1-0:response-profile-pac": {
+                "response-profile-capability": {
+                  "operation-name": "/v1/inform-about-release-history-in-generic-representation",
+                  "field-name": {
+                    "field-name-reference": "/core-model-1-4:control-construct/logical-termination-point=okm-2-0-0-http-s-bm-000/layer-protocol=0/http-server-interface-1-0:http-server-interface-pac/http-server-interface-capability/release-list=0/release-number"
+                  },
+                  "description": "List of registered application names and release numbers",
```
Comment (IswaryaaS) [inline OperationKeyManagement+config.json:224]: description in line 224 (okm-2-0-0-response-p-007) and 243 (okm-2-0-0-response-p-008) to be changed to "List of release numbers and changes"
path : GenericResponseProfile :: /v1/inform-about-release-history-in-generic-representation :: capability :: description

File: OperationKeyManagement+profileInstances.yaml [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+      datatype: 'string'
+    configuration:
+      value-reference: '/core-model-1-4:control-construct/logical-termination-point=okm-2-0-0-http-s-bm-000/layer-protocol=0/http-server-interface-1-0:http-server-interface-pac/http-server-interface-capability/owner-email-address'
+
+  - profile-name: 'GenericResponseProfile'
+    uuid: 'okm-2-0-0-response-p-007'
+    capability:
+      operation-name: '/v1/inform-about-release-history-in-generic-representation'
+      field-name-reference: '/core-model-1-4:control-construct/logical-termination-point=okm-2-0-0-http-s-bm-000/layer-protocol=0/http-server-interface-1-0:http-server-interface-pac/http-server-interface-capability/release-list=0/release-number'
+      description: 'List of registered application names and release numbers'
```
Comment (IswaryaaS) [inline OperationKeyManagement+profileInstances.yaml:115]: description in line 115 (okm-2-0-0-response-p-007) and 125 (okm-2-0-0-response-p-008) to be changed to "List of release numbers and changes"
path : GenericResponseProfile :: /v1/inform-about-release-history-in-generic-representation :: capability :: description

**Review by IswaryaaS — CHANGES_REQUESTED** [review]

File: OperationKeyManagement+config.json (hunk 2 above)
Comment (openBackhaul) [reply OperationKeyManagement+config.json:224]: 224: Took the value from the ApplicationPattern ("Represents the first entry in the list of releases"). Many thanks for indicating.
243: Changed to "List of release numbers and changes".

File: OperationKeyManagement+profileInstances.yaml (hunk 3 above)
Comment (openBackhaul) [reply OperationKeyManagement+profileInstances.yaml:115]: 115: Took the value from the ApplicationPattern ("Represents the first entry in the list of releases"). Many thanks for indicating.
125 Changed as proposed.

File: OperationKeyManagement+config.json (hunk 1 above)
Comment (openBackhaul) [reply OperationKeyManagement+config.json:77]: Changed as proposed.

**Review by IswaryaaS — DISMISSED** [review]
Changes looks fine for config.json, ProfileInstances.yaml, profiles.yaml, services.yaml files.
Note : OperationKeyManagement.yaml file is about to be updated with profile related data.

**Pull request merged**